modin/engines/base/frame/data.py

```diff
@@ -1844,8 +1834,28 @@ def _binary_op(self, op, right_frame, join_type="outer"):
         BasePandasFrame
             A new dataframe.
         """
-        left_parts, right_parts, joined_index = self._copartition(
-            0, right_frame, join_type, sort=True
+        joined_index, ilidx, iridx = self.axes[0].join(
```

Comment:
Why `ilidx` and `iridx` are started with `i`? What does it mean? Just curious.

Reply:
In the context of a pandas, this meant that the indexer refers to df.index (not to df.columns). Renamed.